source/uwp/Renderer/idl/AdaptiveCards.Rendering.Uwp.idl

```diff
@@ -455,6 +455,7 @@ namespace AdaptiveCards { namespace Rendering { namespace Uwp
     {
         [propget] HRESULT AdaptiveCard([out, retval] AdaptiveCard** value);
         [propget] HRESULT Errors([out, retval] Windows.Foundation.Collections.IVector<IAdaptiveError*>** value);
+        [propget] HRESULT Warnings([out, retval] Windows.Foundation.Collections.IVector<IAdaptiveWarning*>** value);
```

Comment:
>IVector [](start = 80, length = 7)

Should this be IVectorView? It doesn't seem like you would want callers to modify the vector.

Reply:
Good point, but we use the interface internally also to populate the vector. The changes to fix this would be out of scope of this change, errors is the same way.